### PR TITLE
Fix: modius configs

### PR DESCRIPTION
**Pull request by OjusWiZard**
## Proposed changes
- [x] Fix the `fixed` provision-type environment variables for modius
- [x] Support the `computed` provision-type environment variables for modius

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by Tanya-atatakai — APPROVED** [review]

File: frontend/constants/serviceTemplates.ts [hunk 1]
```diff
@@ -358,8 +358,8 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
       STAKING_ACTIVITY_CHECKER_CONTRACT_ADDRESS: {
         name: 'Staking activity checker contract address',
         description: '',
-        value: '',
-        provision_type: EnvProvisionType.COMPUTED,
+        value: '0x07bc3C23DbebEfBF866Ca7dD9fAA3b7356116164',
+        provision_type: EnvProvisionType.FIXED,
```
Comment (0xArdi) [inline frontend/constants/serviceTemplates.ts:362]: is this just temporary? a quick solution?

**Review by 0xArdi — APPROVED** [review]

File: frontend/constants/serviceTemplates.ts (hunk 1 above)
Comment (OjusWiZard) [reply frontend/constants/serviceTemplates.ts:362]: I found it hardcoded in the modius-quickstart, so kept it fixed here also. Should it be computed? If yes, then how?

Comment (OjusWiZard) [reply frontend/constants/serviceTemplates.ts:362]: merging right now, to unblock Tanya. But lmk

Comment (0xArdi) [reply frontend/constants/serviceTemplates.ts:362]: The staking contract should define it I think.

**Pull request merged**